components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.m

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCActionSheetColorThemer.h"
+#import "MaterialActionSheet.h"
+
+static const CGFloat primaryAlpha = 0.87f;
+static const CGFloat secondaryAlpha = 0.6f;
```

Comment:
I realize you've gotten conflicting input on this, but the style guide is pretty clear on this: we can and do use `k` prefixes for static consts in our files.

http://go/objc-style#Constants

```
A lowercase k can be used as a standalone prefix for constants of static storage duration declared within implementation files:

static const int kFileCount = 12;
static NSString *const kUserKey = @"kUserKey";
```

Reply:
Done